### DISCUSSION TITLE
Hide one of my column headers

### DISCUSSION THREAD

**Discussion by mdodge-ecgrow** [TanStack/table#5117] [Q&A]
I have a table build with react-table.  Now I want to add a column with the three vertical dots like you sometimes see to give you a few options on that specific row.  For that column, I do not want a header.  I could not find any options in the docs to hide or disable a certain header.  How can I do that?

**Comment by KevinVandy** [comment]
use a custom `header` column option with the jsx that you want?

**Reply by mdodge-ecgrow** [reply]
I'm not sure how to do a custom header column.
